build.rs

```diff
@@ -364,11 +386,23 @@ struct Target {
     arch: String,
     os: String,
     env: String,
-    is_git: bool,
+
+    // Is this a debug build? This affects whether assertions might be enabled
```

Comment:
Nit: This (and the one below) could be made into a doc comment so IDE hover actions show the text without needing to jump to the field definition.

Reply:
Thanks. i made that change!